pkg/slack/modals/bug/view_test.go

```diff
@@ -0,0 +1,164 @@
date":"","initial_option":{"text":null,"value":""},"initial_user":"","selected_channel":"","selected_channels":null,"selected_conversation":"","selected_conversations":null,"selected_date":"","selected_option":{"text":null,"value":""},"selected_options":null,"selected_user":"","selected_users":null,"text":{"text":"","type":""},"type":"plain_text_input","value":"input"}},"impact":{"+2dWl":{"action_id":"","action_ts":"","block_id":"","initial_channel":"","initial_conversation":"","initial_date":"","initial_option":{"text":null,"value":""},"initial_user":"","selected_channel":"","selected_channels":null,"selected_conversation":"","selected_conversations":null,"selected_date":"","selected_option":{"text":null,"value":""},"selected_options":null,"selected_user":"","selected_users":null,"text":{"text":"","type":""},"type":"plain_text_input","value":"input"}},"optional":{"DCgA":{"action_id":"","action_ts":"","block_id":"","initial_channel":"","initial_conversation":"","initial_date":"","initial_option":{"text":null,"value":""},"initial_user":"","selected_channel":"","selected_channels":null,"selected_conversation":"","selected_conversations":null,"selected_date":"","selected_option":{"text":null,"value":""},"selected_options":null,"selected_user":"","selected_users":null,"text":{"text":"","type":""},"type":"plain_text_input","value":""}},"reproduction":{"Cv9":{"action_id":"","action_ts":"","block_id":"","initial_channel":"","initial_conversation":"","initial_date":"","initial_option":{"text":null,"value":""},"initial_user":"","selected_channel":"","selected_channels":null,"selected_conversation":"","selected_conversations":null,"selected_date":"","selected_option":{"text":null,"value":""},"selected_options":null,"selected_user":"","selected_users":null,"text":{"text":"","type":""},"type":"plain_text_input","value":"input"}},"symptom":{"G=Vl":{"action_id":"","action_ts":"","block_id":"","initial_channel":"","initial_conversation":"","initial_date":"","initial_option":{"text":null,"value":""},"initial_user":"","selected_channel":"","selected_channels":null,"selected_conversation":"","selected_conversations":null,"selected_date":"","selected_option":{"text":null,"value":""},"selected_options":null,"selected_user":"","selected_users":null,"text":{"text":"","type":""},"type":"plain_text_input","value":"input"}},"title":{"EU7e8":{"action_id":"","action_ts":"","block_id":"","initial_channel":"","initial_conversation":"","initial_date":"","initial_option":{"text":null,"value":""},"initial_user":"","selected_channel":"","selected_channels":null,"selected_conversation":"","selected_conversations":null,"selected_date":"","selected_option":{"text":null,"value":""},"selected_options":null,"selected_user":"","selected_users":null,"text":{"text":"","type":""},"type":"plain_text_input","value":"input"}}}},"submit":{"emoji":true,"text":"Submit","type":"plain_text"},"team_id":"T01B37EA9T5","title":{"emoji":true,"text":"File a Bug","type":"plain_text"},"type":"modal"}}`),
```

Comment:
Ugh. How useful will these be in practice?

Reply:
I mean, if/when I need these tests to understand how the input affects the output, I am screwed, because I have no idea how to find the _valid because specific component chosen_ in that JSON. Maybe we should build these inputs dynamically from a template if possible?